app/code/Magento/Sales/Test/Mftf/ActionGroup/AdminOpenOrdersPageActionGroup.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminOpenOrdersPageActionGroup">
```

Comment:
Wouldn't it be better to have `AdminOrdersPageOpenActionGroup`? This way all other Action Groups related to Orders Page would be close in list to each other, even in Autosuggestion in PHPStorm:

Eg. when you start to type `AdminOrderPage` and you see hints:
- `OpenActionGroup`
- `ClickFirstActionGroup`
etc. :)